app/policies/user_policy.rb

```diff
@@ -33,4 +33,8 @@ def update?
   def destroy?
     @current_user.superadmin? && !@user.superadmin?
   end
+
+  def reset_password?
+    @current_user.superadmin? && @current_user != @user
```

Comment:
Not sure if this matches the requirement:

> Als Superuser möchte ich Passwort von anderen Superusern resetten (oder wenn einfacher überschreiben)

Shouldn't it be `@current_user.superadmin? || @current_user == @user`, i.e. superadmins can reset all passwords, other users only their own?

And yes this definitely needs a test :)